options.php

```diff
@@ -124,7 +124,11 @@ function wpmautic_base_url() {
  * Define the input field for Mautic script location
  */
 function wpmautic_script_location() {
-	$position = wpmautic_option( 'script_location', '' );
+	$position     = wpmautic_option( 'script_location', '' );
+	$allowed_tags = array(
+		'br'   => [],
```

Comment:
Here, it's better to use the `array()` syntax because we are compatible with very old PHP versions...